src/variables/body/variablesBody.ts

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Jupyter Development Team.
```

Comment:
I would suggest just calling these `Body` and `Description` and renaming the file to `index.ts`